tests/Contributor/ContributorRepositoryTest.php

```diff
@@ -146,4 +147,50 @@ public function it_can_overwrite_contributor(): void
             $this->contributorRepository->getContributors($this->ghentEvent)
         );
     }
+
+    /**
+     * @test
+     * @dataProvider itemTypeDataProvider
+     */
+    public function it_saves_the_correct_itemType(ItemType $itemType): void
```

Comment:
No need to do this in a separate unit test, you can do it inside the existing `it_can_overwrite_contributor`.
There is basically no difference between `it_can_overwrite_contributor` and `it_saves_the_correct_itemType`

Reply:
It thought it would be better to have 1 test that focuses on the updating of an item that has contributors, and a separate one that focuses on seeing that it correctly saves the type. But I can indeed put those 2 in 1 test.

Reply:
@JonasVHG with this change you only test the item type of the first contributor, but all 3 rows should have item type event. Maybe you can use a query builder and create a query with the UUID and the item type. When doing that you only need to check the returned number of rows and that should be 3.